homeassistant/components/zwave_js/fan.py

```diff
@@ -225,3 +236,124 @@ def zwave_speed_to_percentage(self, zwave_speed: int) -> int:
         # the UI handles steps e.g., for a 3-speed fan, you get steps at 33,
         # 67, and 100.
         return round(percentage)
+
+
+class ZwaveThermostatFan(ZWaveBaseEntity, FanEntity):
+    """Representation of a Z-Wave thermostat fan."""
+
+    _fan_mode: ZwaveValue
+    _fan_off: ZwaveValue | None = None
+    _fan_state: ZwaveValue | None = None
+
+    def __init__(
+        self, config_entry: ConfigEntry, client: ZwaveClient, info: ZwaveDiscoveryInfo
+    ) -> None:
+        """Initialize the thermostat fan."""
+        super().__init__(config_entry, client, info)
+
+        self._fan_mode = self.get_zwave_value(
+            THERMOSTAT_FAN_MODE_PROPERTY,
+            CommandClass.THERMOSTAT_FAN_MODE,
+            add_to_watched_value_ids=True,
+            check_all_endpoints=True,
+        )
+
+        if not self._fan_mode:
+            raise ValueError("Thermostat fan mode is required")
+
+        self._fan_off = self.get_zwave_value(
+            THERMOSTAT_FAN_OFF_PROPERTY,
+            CommandClass.THERMOSTAT_FAN_MODE,
+            add_to_watched_value_ids=True,
+            check_all_endpoints=True,
+        )
+        self._fan_state = self.get_zwave_value(
+            THERMOSTAT_FAN_STATE_PROPERTY,
+            CommandClass.THERMOSTAT_FAN_STATE,
+            add_to_watched_value_ids=True,
+            check_all_endpoints=True,
+        )
+
+    async def async_turn_on(
+        self,
+        speed: str | None = None,
+        percentage: int | None = None,
+        preset_mode: str | None = None,
+        **kwargs: Any,
+    ) -> None:
+        """Turn the device on."""
+        if self._fan_off:
+            await self.info.node.async_set_value(self._fan_off, False)
+
+    async def async_turn_off(self, **kwargs: Any) -> None:
+        """Turn the device off."""
+        if self._fan_off:
+            await self.info.node.async_set_value(self._fan_off, True)
+
+    @property
+    def is_on(self) -> bool:
```

Comment:
```suggestion
    def is_on(self) -> bool | None:
```
I don't think you can assume a state if the value is unavailable.